packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -389,6 +410,21 @@ const App = (props) => {
                     </Box>
                 </CurrencyProvider>
             </IntlProvider>
+            {ACTIVE_DATA_ENABLE == 1 && (
+                <script
+                    type="text/javascript"
+                    src={getAssetUrl('static/dwanalytics-22.2.js')}
+                    async="async"
+                    onLoad={trackPage}
+                ></script>
+            )}
+            {ACTIVE_DATA_ENABLE == 1 && (
```

Comment:
After converting to `ACTIVE_DATA_ENABLE = true` we can avoid the suboptimal usage of `==` which is not a good comparator in javascript

```suggestion
            {ACTIVE_DATA_ENABLE && (
```